app/views/spree/shared/payment/_bolt.html.erb

```diff
@@ -3,6 +3,7 @@
   data-payment-method-id="<%= payment_method.id %>"
   data-frontend="<%= frontend %>"
   data-bolt-user-signed-in="<%= spree_current_user&.user_authentications&.where(provider: 'bolt')&.exists? == true %>"
+  data-user-id="<%= spree_current_user&.id %>"
```

Comment:
Providing it from the outside exposes a vulnerability issue.
The user is logged in, we should retrieve the user from the backend.

Reply:
removed